guides/common/modules/proc_migrating-ansible-content.adoc

```diff
@@ -0,0 +1,34 @@
+[id="migrating-ansible-content_{context}"]
+= Migrating Ansible Content
+
+The upgrade from {RHEL} 7 to {RHEL} 8 includes an upgrade of Ansible core from 2.9 to 2.12.
```

Comment:
What's more correct to state, and important, is that it was Ansible Engine and now it's Ansible Core and those two are not equivalent. https://www.redhat.com/en/blog/updates-using-ansible-rhel-86-and-90 is a good reference